packages/react-app/src/services/backend.ts

```diff
@@ -76,6 +77,11 @@ class BackendService {
     return giveaways;
   };
 
+
+  saveGiveaway = (data: FormData) => data.get("_id")
+    ? this.makeRequest(`/giveaways/${data.get('_id')}`, "PUT", { "Content-Type": "multipart/form-data" }, data)
+    : this.makeRequest('/giveaways/', "POST", { "Content-Type": "multipart/form-data" }, data);
```

Comment:
Here we could have the ternary only on the method type, I think would be more clear